extensions/Push/lib/Connector/Webhook.pm

```diff
@@ -76,7 +77,7 @@ sub should_send {
   {
     if ($event =~ /create/ && $message->routing_key eq 'bug.create') {
       return 1;
-    }elsif ($event =~ /change/ && $message->routing_key =~ /\Qbug.modify\E/) {
+    }elsif ($event =~ /change/ && ($message->routing_key =~ /\Qbug.modify\E/ || $message->routing_key eq 'comment.create')) {
```

Comment:
\Q\E are not needed since we are looking for an exact string so let's go ahead and  make the line:

```perl
}elsif ($event =~ /change/ && ($message->routing_key =~ /^bug\.modify/ || $message->routing_key eq 'comment.create')) {
```